Jenkinsfile

```diff
@@ -332,6 +348,35 @@ pipeline {
                 checkout scm
             }
         }
+        stage('check_for_changed_files') {
+          steps {
+            script {
+	      if (env.BRANCH_NAME == 'master') {
+	        return
+	      }
+
+	      def all_files = new HashSet<String>()
+
+              for (set in currentBuild.changeSets) {
+                def entries = set.items
+                for (entry in entries) {
```

Comment:
Do we need an intermediate, or is `for (entry in set.items)` good enough?

Reply:
I have no idea here about what groovy need/wants/balks at.

Reply:
Lets leave it in for now. Does not hurt I believe.